Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```diff
@@ -14,7 +14,7 @@
       tags:
         - CartridgeLightRifle
     proto: CartridgeLightRifle
-    capacity: 30
+    capacity: 23
```

Comment:
23 feels like a very odd number - i know this is to keep the per-magazine damage in-line with previous values, but it feels strange to me

Reply:
I'm open to ideas for this. If I round its number up to 25, the total magazine damage becomes 625 which is, a LOT. Like WAY more than anything Security has. I could bump it down to 20 to be in-line with the Daito, which is okay I guess. 23 just happens to be that weird prime number comfort zone for number of shots vs. total damage.

Reply:
yeah it's a weird situation. i'll just take this for now and we'll see about adjustments later (probably) (we are going to forget)